src/microbe_stage/CellHexPhotoBuilder.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Godot;
+
+public class CellHexPhotoBuilder : Spatial, IPhotographable
```

Comment:
```suggestion
public class CellHexesPhotoBuilder : Spatial, IPhotographable
```
I think that kind of name would be a bit clearer that it's not a "cell hex" but "cell hexes" (the hexes the cell consists of) photo builder.